pygpudrive/env/env_torch.py

```diff
@@ -227,6 +245,47 @@ def normalize_ego_state(self, state):
 
         return state
 
+    def get_expert_actions(self, debug_world_idx=None, debug_veh_idx=None):
+        """Get expert actions for the full trajectories across worlds."""
+        expert_traj = self.sim.expert_trajectory_tensor().to_torch()
+        positions = expert_traj[:, :, :2 * self.episode_len].view(self.num_worlds,
+                                                                  self.max_agent_count,
+                                                                  self.episode_len, -1)
+
+        velocity = expert_traj[:, :, 2 * self.episode_len:4 * self.episode_len].view(self.num_worlds,
+                                                                                     self.max_agent_count,
+                                                                                     self.episode_len, -1)
+        if self.action_type == 'delta':
+            inferred_expert_actions = expert_traj[:, :, -3 * self.episode_len:].view(self.num_worlds,
+                                                                                     self.max_agent_count,
+                                                                                     self.episode_len, -1)
+            inferred_expert_actions[..., 0] = torch.clamp(inferred_expert_actions[..., 0], -6, 6)
+            inferred_expert_actions[..., 1] = torch.clamp(inferred_expert_actions[..., 1], -6, 6)
+            inferred_expert_actions[..., 2] = torch.clamp(inferred_expert_actions[..., 2], -3.14, 3.14)
+        else:
+            inferred_expert_actions = expert_traj[:, :, -6 * self.episode_len:-3 * self.episode_len].view(
+                self.num_worlds,
+                self.max_agent_count,
+                self.episode_len, -1)
+            inferred_expert_actions[..., 0] = torch.clamp(inferred_expert_actions[..., 0], -6, 6)
+            inferred_expert_actions[..., 1] = torch.clamp(inferred_expert_actions[..., 1], -0.3, 0.3)
+        velo2speed = None
+        debug_positions = None
+        if debug_world_idx is not None and debug_veh_idx is not None:
```

Comment:
Small comment: If you want support flexible logging, a better practice is to use the `logging` module (e.g. logging.debug() can be used here instead of providing the `debug_world_idx` argument)

https://docs.python.org/3/library/logging.html#module-logging